pkg/deploy/strategy/support/lifecycle.go

```diff
@@ -159,6 +163,7 @@ func makeHookPod(hook *deployapi.LifecycleHook, deployment *kapi.ReplicationCont
 				},
 			},
 			ActiveDeadlineSeconds: &maxDeploymentDurationSeconds,
+			NodeSelector:          nodeSelector,
```

Comment:
Could just do the assignmen/comment inline since the local variable isn't used anywhere else.